examples/react-app/src/index.tsx

```diff
@@ -82,14 +83,16 @@ ReactDOM.render(
         }}
       >
         <span>Menu Item</span>
-
         <PlmgSeparator
           thickness="thick"
           style={{ alignSelf: 'stretch' }}
           direction="vertical"
           color="#78909C"
         ></PlmgSeparator>
-        <span>Menu Item 2</span>
+        <span>Menu Item 2</span>{' '}
+        <PlmgStatus variant={'foo'} iconLeft={'duck'}>
```

Comment:
Nonsense varient.
This does not help show off the component and it does not encourage me that including validation on render is of benefit if you are not even seeing the error message you are enforcing.
```suggestion
        <PlmgStatus variant={'neutral'} iconLeft={'duck'}>
```

Reply:
Yes, I was demonstrating the error message to Jake and didn't revert it before committing. Should have looked at the diff and not assumed.

Enforcing the validation on render does at least produce a console error, but yes the component does still render using the invalid props. This is happening across the plumage library except without us seeing an error message until the component re-renders.

![Screenshot 2022-08-18 at 08 51 47](https://user-images.githubusercontent.com/104433221/185320547-975dbf29-5752-4868-b496-3a1da6e33097.png)

Reply:
Better example added.

Reply:
> Enforcing the validation on the initial render produces a console error. Currently this is happening across the plumage library except without us seeing an error message until the component re-renders. I don't think calling the validate functions on initial render causes performance issues. As I've said, I see no point in validation functions that only trigger on re-render (unless I'm misunderstanding the point of these validation functions.)

I'll let you take this discussion to the Plumage team to define guidelines for the rest of us. 🙏